src/packet_analysis/protocol/gre/GRE.cc

```diff
@@ -82,11 +82,7 @@ bool GREAnalyzer::AnalyzePacket(size_t len, const uint8_t* data, Packet* packet)
 			{
 			// transparent ethernet bridging
 			if ( len > gre_len + 14 )
-				{
-				eth_len = 14;
```

Comment:
Looks like `eth_len` is still used below in line 203, would't it have an unexpected value there after this is removed?

Reply:
We must not have a test that exercises that, because that shouldn't have worked. Nice catch. I fixed it.